src/app/frame.ts

```diff
@@ -1,13 +1,13 @@
-import { app, debug } from "../kaplay";
+import { _k } from "../kaplay";
 
 export function dt() {
-    return app.dt() * debug.timeScale;
```

Comment:
debug.timeScale is no longer needed?

Reply:
no because app.dt() already did it, it was dupplied

Reply:
😱